tests/data_files/mock_rbac_returns_multiple_read_permissions.json

```diff
@@ -0,0 +1,75 @@
+{
+  "meta": {
+    "count": 5,
+    "limit": 1000,
+    "offset": 0
+  },
+  "links": {
+    "first": "/api/rbac/v1/access/?application=inventory&limit=1000&offset=0",
+    "next": null,
+    "previous": null,
+    "last": "/api/rbac/v1/access/?application=inventory&limit=1000&offset=0"
+  },
+  "data": [
+    {
+      "resourceDefinitions": [],
+      "permission": "ros:*:*"
+    },
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "group.id",
+            "value": [
+              "12345678-fe1b-4191-8408-cbadbd47f7a3"
+            ],
+            "operation": "in"
+          }
+        }
+      ],
+      "permission": "inventory:groups:read"
+    },
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "group.id",
+            "value": [
+              "155860d5-648c-4529-847a-690cbf198934"
+            ],
+            "operation": "in"
+          }
+        }
+      ],
+      "permission": "inventory:groups:write"
+    },
+    {
+      "resourceDefinitions": [
+        {
+          "attributeFilter": {
+            "key": "group.id",
+            "value": [
+              "abcdefgh-d97e-4ed0-9095-ef07d73b4839"
+            ],
+            "operation": "in"
+          }
+        }
+      ],
+      "permission": "inventory:hosts:read"
```

Comment:
@upadhyeammit I observed there is a mock RBAC file `mock_rbac_returns_groups_including_example_group` in which the value for `inventory:hosts:read` permission contains multiple value but based on current RBAC implementation it is single value. Can we also update that file please to avoid confusion in future?

Reply:
and I feel this will also help in making unit test better. let me know your thoughts 🙌🏻